analyzer-models/src/test/java/ffb/analyzer/models/espn/PlayerRankingTests.java

```diff
@@ -0,0 +1,47 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import ffb.analyzer.models.espn.PlayerRanking.RankType;
+import ffb.analyzer.models.espn.deserializers.PlayerRankingsDeserializer;
+import java.io.IOException;
+import java.util.List;
+import org.junit.Assert;
+
+/**
+ * Unit tests for a {@link PlayerRanking}.
+ */
+public class PlayerRankingTests extends DeserializingResourceLoader {
+
+    @Override
+    protected ObjectMapper configureObjectMapper() {
+        SimpleModule module = new SimpleModule().addDeserializer(List.class, new PlayerRankingsDeserializer());
+
+        return super.configureObjectMapper()
+            .registerModule(module);
+
+    }
+
+    @Override
+    public void testDeserialization() throws IOException {
+        List<PlayerRanking> rankings = deserializeObjects(PlayerRanking.class);
+
+        Assert.assertEquals(4, rankings.size());
+        rankings.forEach(this::testRankingDeserialization);
+    }
+
+    private void testRankingDeserialization(PlayerRanking ranking) {
+        Assert.assertEquals(0, ranking.getAuctionValue());
+        Assert.assertEquals(0, ranking.getRank());
+        Assert.assertEquals(0, ranking.getRankSourceId());
+        Assert.assertEquals(4, ranking.getSlotId());
+        Assert.assertTrue(
```

Comment:
What's the benefit of this assertion if we're checking for every possible enum value?